content/scout/image-analysis.md

```diff
@@ -0,0 +1,208 @@
+3. Select one of your local images in the list.
+
+   This opens the [Image details view](./image-details-view.md), showing a
+   breakdown of packages and vulnerabilities found by the Docker Scout analysis
+   for the image you selected.
+
+### CLI
+
+The `docker scout` CLI commands provide a terminal interface for using Docker
+Scout with local and remote images.
+
+Using the `docker scout quickview` and `docker scout cves` CLI commands, you
+can analyze images locally and view the analysis report in text format. You can
+print the results directly to stdout, or export them to a file using a
+structured format, such as Static Analysis Results Interchange Format (SARIF).
+
+#### Install
+
+The Docker Scout CLI plugin is available in Docker Desktop starting with
+version 4.17 and available as a standalone binary.
+
+To install the latest version of the plugin manually, run the following
+commands:
+
+```console
+$ curl -fsSL https://raw.githubusercontent.com/docker/scout-cli/main/install.sh -o install-scout.sh
+$ sh install-scout.sh
+```
+
+> **Note**
+>
+> Always examine scripts downloaded from the internet before running them
+> locally. Before installing, make yourself familiar with potential risks and
+> limitations of the convenience script.
+
+If you want to install the plugin manually, you can find full instructions in
+the [plugin's repository](https://github.com/docker/scout-cli).
+
+The plugin is also available as [a container image](https://hub.docker.com/r/docker/scout-cli)
+and as [a GitHub action](https://github.com/docker/scout-action).
+
+#### Quickview
+
+The `quickview` command provides an overview of the vulnerabilities found in a
+given image and its base image.
+
+```console
+$ docker scout quickview traefik:latest
+    ✓ SBOM of image already cached, 311 packages indexed
+
+  Your image  traefik:latest  │    0C     2H     8M     1L
+  Base image  alpine:3        │    0C     0H     0M     0L
+```
+
+If your the base image is out of date, the `quickview` command also shows how
+updating your base image would change the vulnerability exposure of your image.
+
+```console
+$ docker scout quickview postgres:13.1
+    ✓ Pulled
+    ✓ Image stored for indexing
+    ✓ Indexed 187 packages
+
+  Your image  postgres:13.1                 │   17C    32H    35M    33L
+  Base image  debian:buster-slim            │    9C    14H     9M    23L
+  Refreshed base image  debian:buster-slim  │    0C     1H     6M    29L
+                                            │    -9    -13     -3     +6
+  Updated base image  debian:stable-slim    │    0C     0H     0M    17L
+                                            │    -9    -14     -9     -6
+```
+
+#### CVEs
+
+The `cves` command gives you a complete view of all the vulnerabilities in the
+image. This command supports several flags that lets you specify more precisely
+which vulnerabilities that you're interested in. For example, by severity or by
+package type:
```

Comment:
"For example, by serverity or by package type" is not a sentence.

Suggest: This command supports several flags that lets you specify more precisely which vulnerabilities you are interested in, e.g., by severity or package type:

Reply:
Thanks for the suggestion. I will take it - except I avoid latin abbreviations since our audience is international and may not be comfortable with them.